README.md

```diff
@@ -1,3 +1,48 @@
 # Akvorado top ASN stats
 
-**WIP** This repository contains some scripts to fetch the top ASN traffic from a clickhouse instance containing [Akvorado](https://github.com/akvorado/akvorado) flow data.
+This repository contains some scripts to fetch the top ASN traffic from a clickhouse instance containing [Akvorado](https://github.com/akvorado/akvorado) flow data.
+
+## Configuration
```

Comment:
I would prefer to split the clickhouse export description from the other scripts in this repository. Let's move most of this content to `akvo-top-asn/README.md` and add only a small overview & toc in the top level.